data/presets/natural/stone.json

```diff
@@ -13,7 +19,10 @@
     "terms": [
         "boulder",
         "stone",
-        "rock"
+        "rock",
+        "findling",
+        "glacial erratic",
+        "erratic"
     ],
-    "name": "Unattached Stone / Boulder"
+    "name": "Glacial Erratic / Boulder"
```

Comment:
It’s been a long time since my geology appreciation class in school. “Glacial erratic” is a term I’ve never encountered before, despite hailing from in a region where terms like “terminal moraine” are well known. If there’s no colloquial term for this specific concept, then “glacial erratic” is a good name for the preset. But seeing “boulder” next to it might make me inclined to choose this obscurely named preset over “Rock” when tagging a decorative boulder. Maybe “Boulder” should be an alias but not part of the name? (“Decorative boulders”, marketed as such, are so common at garden stores that they could even be a term for that preset.)

Reply:
I was kind of on the fence since boulder is a more commonly known term. The German concept of "findling" that the tag is used for does seem to be a near exact match to "glacial erratic" though so boulder could just be an alias. Good point about decorative boulders / landscape boulders, I was thinking a lot of what people would describe as boulders count as glacial erratics anyway but those would not.

Thank you for the feedback, I will update 👍